core/client/src/test/java/alluxio/client/block/policy/options/CreateOptionsTest.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.client.block.policy.options;
+
+import alluxio.CommonTestUtils;
+import alluxio.util.CommonUtils;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.Random;
+
+/**
+ * Tests for the {@link CreateOptions} class.
+ */
+public final class CreateOptionsTest {
+  /**
+   * Tests for defaults {@link CreateOptions}.
+   */
+  @Test
+  public void defaults() throws IOException {
+    CreateOptions options = CreateOptions.defaults();
+
```

Comment:
good question @jswudi . `defaults()` is a factory method here to create a new instance. This `CreateOptions` class, like many other `FooOptions` class in Alluxio codebase, is a way to specify arguments in `create` API. Thus it has a strong pattern---typically we do something like this `CreateOptions.defaults().setFoo(1).setBar(2)`, to customize non-default options. In this scenario, `CreateOptions.defaults()` is nicer than `new CreateOptions()` as it is more obvious the instance you get represents the default value